odl/trafos/fourier.py

```diff
@@ -2002,10 +2057,10 @@ def __init__(self, ran, dom=None, impl='numpy', **kwargs):
             of frequencies. It is guaranteed to be one-to-one
             (invertible).
 
-        * The `Operator.domain` of this operator always has the
+        * The `Operator.range` of this operator always has the
           `ComplexNumbers` as `LinearSpace.field`, i.e. if the
-          field of ``ran`` is the `RealNumbers`, this operator has no
-          `Operator.adjoint`.
+          field of ``dom`` is the `RealNumbers`, this operators adjoint is
+          defined by identifying real and complex parts.
```

Comment:
This makes no sense. You mean "defined by identifying real and imaginary parts with the components of a real product space element."
We should also have a section in the math documentation so we can refer to it here.